jans-cedarling/bindings/cedarling_python/tests/test_authorize_multi_issuer.py

```
@@ -0,0 +1,225 @@
+    {
+        "cedar_entity_mapping": {"entity_type": "Jans::Issue", "id": "random_id"},
+        "org_id": "some_long_id",
+        "country": "US",
+    }
+)
+
+
+def get_multi_issuer_config():
+    """Helper to configure Cedarling for multi-issuer tests"""
+
+    def configure_for_multi_issuer(config):
```

Ruff (0.14.3)

Missing return type annotation for private function `configure_for_multi_issuer`

Add return type annotation: `None`

(ANN202)